src/views/pages/_index.html

```
@@ -93,14 +93,50 @@
-			<div class="gp_map-block_header">
-				<span></span>
-				<span></span>
-				<span></span>
+		{{#if focusModeAvailable || focusModeActive}}
+			<div
+				class="gp_focus-mode-backdrop {{#if focusModeActive}}active{{/if}}"
+				aria-hidden="true"
+				on-click="@this.closeFocusModeFromBackdrop(@event)"></div>
+		{{/if}}
+
+		<div
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)